src/main/java/core/basesyntax/vehicles/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.vehicles;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Excavator is working");
```

Comment:
The message in the `doWork` method should indicate the type of machine more informatively. Consider including the class name in the message.